src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx

```diff
@@ -431,7 +436,7 @@ You can use keyboard shortcuts to navigate through the UI efficiently.
                 Open global search
             </td>
             <td>
-                `/`
+                Press `K` while holding `Ctrl` + `Cmd` 
```

Comment:
Is there a windows equivalent? Looks like some of these alternate between only giving the MacOS shortcuts, only the Windows, or both.

Reply:
Windows equivalent == Ctrl + K (source: big doc you linked!)